tests/core/math/test_transform_2d.h

```diff
@@ -45,6 +45,90 @@ Transform2D identity() {
 	return Transform2D();
 }
 
+TEST_CASE("[Transform2D] Default constructor") {
+	Transform2D default_constructor = Transform2D();
+	CHECK(default_constructor == Transform2D(Vector2(1, 0), Vector2(0, 1), Vector2(0, 0)));
+}
+
+TEST_CASE("[Transform2D] Copy constructor") {
+	Transform2D T = create_dummy_transform();
+	Transform2D copy_constructor = Transform2D(T);
+	CHECK(T == copy_constructor);
+}
+
+TEST_CASE("[Transform2D] Constructor from angle and position") {
+	const float ROTATATION = Math_PI / 4;
+	const Vector2 TRANSLATION = Vector2(20, -20);
+
+	Transform2D test = Transform2D(ROTATATION, TRANSLATION);
+	Transform2D expected = Transform2D().rotated(ROTATATION).translated(TRANSLATION);
+	CHECK(test == expected);
+}
+
+TEST_CASE("[transform2d] Constructor from angle, scale, skew and position") {
+	const float ROTATATION = Math_PI / 2;
+	const Vector2 SCALE = Vector2(2, 0.5);
+	const float SKEW = Math_PI / 4;
+	const Vector2 TRANSLATION = Vector2(30, 0);
+
+	Transform2D test = Transform2D(ROTATATION, SCALE, SKEW, TRANSLATION);
+	Transform2D expected = Transform2D().scaled(SCALE).rotated(ROTATATION).translated(TRANSLATION);
+	expected.set_skew(SKEW);
+
+	CHECK(test.is_equal_approx(expected));
+}
+
+TEST_CASE("[Transform2D] Constructor from raw values") {
+	Transform2D test = Transform2D(1, 2, 3, 4, 5, 6);
+	Transform2D expected = Transform2D(Vector2(1, 2), Vector2(3, 4), Vector2(5, 6));
+	CHECK(test == expected);
+}
+
+TEST_CASE("[Transform2D] xform") {
+	Vector2 v = Vector2(2, 3);
+	Transform2D T = Transform2D(Vector2(1, 2), Vector2(3, 4), Vector2(5, 6));
+	Vector2 expected = Vector2(1 * 2 + 3 * 3 + 5 * 1, 2 * 2 + 4 * 3 + 6 * 1);
+	CHECK(T.xform(v) == expected);
+}
+
+TEST_CASE("[Transform2D] Basis xform") {
+	Vector2 v = Vector2(2, 2);
+	Transform2D T1 = Transform2D(Vector2(1, 2), Vector2(3, 4), Vector2(0, 0));
+
+	// Both versions should be the same when the origin is (0,0)
```

Comment:
The comment style is to end comments with a period, see below

Reply:
Ok!